npm/pkg/controlplane/controllers/v1/podController_test.go

```diff
@@ -690,6 +691,88 @@ func TestHasValidPodIP(t *testing.T) {
 	}
 }
 
+func TestIsCompletePod(t *testing.T) {
+	var zeroGracePeriod int64
+	var defaultGracePeriod int64 = 30
+
+	type podState struct {
+		Phase                      corev1.PodPhase
```

Comment:
nitpick: doesn't matter, but is it Go convention to lowercase local struct fields?

Reply:
I am not sure, but I think it is good to have lowercase.

Reply:
it's not convention, the casing of struct fields has the same syntactic meaning as public/private keywords in other languages. 
this is a field in a private type, it should be private (lowercase).

Reply:
So, I addressed it in a right way!